### PR TITLE
Added Jenkins file and made changes to build.gradle file

### PR DESCRIPTION
**Pull request by svodithala-bmj**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -43,10 +50,50 @@ dependencies {
...
+	
+
+dependencyManagement {
+	imports {
+		mavenBom "org.springframework.cloud:spring-cloud-dependencies:${springCloudVersion}"
+	}
+}
+
+jar {
+    baseName = project_name
```
Comment (msepler-bmj) [inline build.gradle]: You need to define the project_name and group variables externally, for example in a gradle.properties file. (like this one: https://github.com/BMJ-Ltd/email-alerts-frontend/blob/master/gradle.properties)

File: build.gradle [hunk 2]
```diff
@@ -43,10 +50,50 @@ dependencies {
...
+    version = version
+    manifest {
+        attributes(
+            "Implementation-Title": project_name,
+            "Implementation-Version": version
+        )
+    }
```
Comment (msepler-bmj) [inline build.gradle]: Also, you need to use the maven publishing plugin in order to send the jar to Nexus successfully. Something like this should work:

```
publishing {
    publications {
        maven(MavenPublication){
            version = version
            artifactId = project_name
            groupId = group
            from components.java
        }
    }
    repositories {
        maven {
            credentials {
                username nexusUsername
                password nexusPassword
            }
            url nexusReleases
        }
    }
}
```

File: build.gradle (hunk 1 above)
Comment (svodithala-bmj) [reply build.gradle]: Added gradle.properties

File: build.gradle (hunk 2 above)
Comment (svodithala-bmj) [reply build.gradle]: Updated the build.gradle file

File: build.gradle [hunk 3]
```diff
@@ -30,6 +32,11 @@ repositories {
 //	springCloudVersion = 'Finchley.RELEASE'
```
Comment (msepler-bmj) [inline build.gradle]: I think this also should be uncommented

**Pull request merged**